packages/mobile/src/screens/app-screen/AppTabNavigationProvider.tsx

```diff
@@ -32,7 +31,7 @@ export const AppTabNavigationProvider = (
   props: AppTabNavigationProviderProps
 ) => {
   const { children } = props
-  const [navigation, setNavigation] = useState<Nullable<AppTabNavigation>>(null)
+  const [navigation, setNavigation] = useState({} as AppTabNavigation)
```

Comment:
Why was this needed?

Reply:
yeah it's pretty bad, but since we are using this as the main navigator now, there is a moment where it's null as it's setting up, which was causing runtime issues. Was thinking this is a bit easier to do here, than cascade the null down to the useNavigate, which was messing with it's returned type, causing useNavigation() to return Nullable<Navigator>, so casted early. lmk though, open to suggestions

Reply:
Oof, yeah this is probably fine for know